tests/src/test/scala/whisk/core/cli/test/WskBasicUsageTests.scala

```diff
@@ -108,6 +108,36 @@ class WskBasicUsageTests
         }
     }
 
+    it should "set valid cert key to get expected success result for client certificate verification" in {
+        val tmpwskprops = File.createTempFile("wskprops", ".tmp")
+        try {
+            val namespace = wsk.namespace.list().stdout.trim.split("\n").last
+            val env = Map("WSK_CONFIG_FILE" -> tmpwskprops.getAbsolutePath())
+            val stdout = wsk.cli(Seq("property", "set", "-i", "--apihost", wskprops.apihost, "--auth", wskprops.authKey,
+                "--cert", wskprops.cert, "--key", wskprops.key, "--namespace", namespace), env = env).stdout
```

Comment:
Adding a comment on how `--namespace` forces an API request and thereby a cert. validation would be helpful to understand the testcase later on.